code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -31,10 +31,13 @@
 			if(istype(M,/mob/living/carbon/human))
 				var/mob/living/carbon/human/H = M
 				if(!H.check_has_mouth())
-					user << "Where do you intend to put \the [src]? You don't have a mouth!"
-					return
-
-			M << "\blue You swallow a gulp of [src]."
+					if(!H.get_species() == "Machine")
+						user << "Where do you intend to put \the [src]? You don't have a mouth!"
+						return
+					else
+						M << "<span class = 'notice'> You pour a bit of liquid from [src] into your connection port.</span>"

```

Comment:
I'd change this to something like fuel or lubricant or liquid-intake port. Connector ports sound purely electronic like a VGA cable connection, and would not transfer liquids into the systems (or shouldn't, at least)